UnityProject/Assets/Scripts/Objects/Construction/WindowObject.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using Mirror;
+
+/// <summary>
+/// The main girder component
+/// </summary>
+public class WindowObject : NetworkBehaviour, ICheckedInteractable<HandApply>, IServerSpawn
+{
+	private TileChangeManager tileChangeManager;
+
+	private RegisterObject registerObject;
+	private ObjectBehaviour objectBehaviour;
+
+	[Tooltip("Layer tile which this will create when placed.")]
+	public LayerTile layerTile;
+	//tracked server side only
+	private int plasteelSheetCount;
```

Comment:
plasteel sheet? you might have some comments and names left from girder/wall

Reply:
Yeah, I saw it too -_-